packages/fulfillment/integration-tests/__tests__/fulfillment-module-service.spec.ts

```diff
@@ -249,6 +249,200 @@ moduleIntegrationTestRunner({
             )
           })
         })
+
+        describe("shipping options", () => {
+          it("should list shipping options with a filter", async function () {
+            const fulfillmentSet = await service.create({
+              name: "test",
+              type: "test-type",
+              service_zones: [
+                {
+                  name: "test",
+                },
+              ],
+            })
+
+            const shippingProfile = await service.createShippingProfiles({
+              name: "test",
+              type: "default",
+            })
+
+            // TODO: change that for a real provider instead of fake data manual inserted data
+            const [{ id: providerId }] =
```

Comment:
NIT: We can have a helper func (inside the .spec file is just fine) with this exact code, as it will be cleaner IMO